### PR TITLE
don't parse ASP.NET websites

### PR DESCRIPTION
**Pull request by jmbeach**
Fixes issue "The given path's format is not supported" #1036 when you have an ASP.NET website in your solution.

### CODE REVIEW

**Comment by jmbeach** [comment]
Travis ci timed out...

File: src/OmniSharp.MSBuild/SolutionParsing/ProjectBlock.cs [hunk 1]
```diff
@@ -43,6 +43,11 @@ private ProjectBlock(string projectTypeGuid, string projectName, string relative
             Sections = sections;
         }
 
+        public bool IsAspWebsite()
+        {
+            return RelativePath.StartsWith("http://");
```
Comment (filipw) [inline src/OmniSharp.MSBuild/SolutionParsing/ProjectBlock.cs]: it should probably be case insensitive. can this be `https://` too? 
btw, do you have a sample project like this? I don't think it's possible to create these types of projects in VS for quite a while now

Comment (jmbeach) [reply src/OmniSharp.MSBuild/SolutionParsing/ProjectBlock.cs]: My project was made in Visual Studio 2013 for my work.

Steps to make one:

- Right click solution item in solution explorer
- Add
    - New website

The option is still there in Visual Studio 2017 Community Edition. Don't know if it works the same.

**Comment by david-driscoll** [comment]
Yay mac builds, going to see if this one passes, if it doesn't I'm sure things change wasn't the culprit and I'll merge anyway.

**Pull request merged**